modelerfour/modeler/modelerfour.ts

```diff
@@ -68,6 +68,13 @@ export class ModelerFour {
     }));
   }
 
+  location(obj: any): string {
+    if (obj['x-ms-metadata'] && obj['x-ms-metadata'].originalLocations) {
```

Comment:
Kindof a nit, but would be nice to not have to pay for the key lookup 3 times

Reply:
yeah, I should use a chaining operator here.